modules/@apostrophecms/express/index.js

```diff
@@ -555,14 +555,20 @@ module.exports = {
           address = false;
         }
 
-        try {
-          await listen();
-        } catch (e) {
-          if (process.env.NODE_ENV !== 'production') {
-            // Retry once, working around a frequently seen timing issue in nodemon
-            await Promise.delay(500);
-            // If it fails again, let it throw
+        const attempts = (process.env.NODE_ENV === 'production') ? 1 : 5;
```

Comment:
Couldn't stand it, added more retries. Working theory is the socket is slow to close because of our long polling.